scripts/CEE/hive-data-collection/script.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+#
+# Collect data from Hive clusters in order to troubleshoot/debug cluster provisioning delays.
+# Input:
+#  (Env) CLUSTERID
+#        Description: The Internal Cluster ID for which data is being collected
+
+set -e
+set -o nounset
+set -o pipefail
+
+start_job(){
+    echo "Job started at $(date +"%Y-%m-%d %T")"
+    echo ".................................."
+    echo
+}
+
+finish_job(){
+    echo
+    echo ".................................."
+    echo "Job finished at $(date +"%Y-%m-%d %T")"
+}
+
+HIVE_NAMESPACE="uhc-production-${CLUSTER_ID}"
+
+if ! oc get namespace "${HIVE_NAMESPACE}" &>/dev/null; then
+    echo "Namespace ${HIVE_NAMESPACE} does not exist. Please check the CLUSTER_ID."
+    exit 1
+fi
+
+## Function which checks the clusterdeployment status
+## and print the details of each clusterdeployment
+clusterdeployment(){
```

Comment:
Can we update the function name to format `verb_resource`? like `read_clusterdeployment()` or `fetch_clusterdeployment()`